cpp/oneapi/dal/io/csv/detail/graph_service.hpp

```diff
@@ -14,17 +13,17 @@
 * See the License for the specific language governing permissions and
 * limitations under the License.
 *******************************************************************************/
-
-/// @file
-/// Io related common data type aliases
-
 #pragma once
 
-#include "oneapi/dal/common.hpp"
-#include "oneapi/dal/detail/common.hpp"
-#include "oneapi/dal/io/detail/common.hpp"
+#include "oneapi/dal/graph/detail/container.hpp"
 
 namespace oneapi::dal::preview {
+
+namespace detail {
```

Comment:
Why do we need edge list container here?

Reply:
it was just migrated from file other file